services/tasks-svc/internal/patient/aggregate/actions.go

```diff
@@ -65,3 +65,11 @@ func (a *PatientAggregate) ReadmitPatient(ctx context.Context) error {
 	}
 	return a.Apply(event)
 }
+
+func (a *PatientAggregate) DeletePatient(ctx context.Context) error {
```

Comment:
I decided to do this, as I think having different (deleted) eventTypes/identifiers for each aggregate makes sense.
We may have an event which technically deletes the aggregate but wants to be identified in another way. For example, instead of `PRESCRIPTION_DELETED`, we may want to use `PRESCRIPTION_ARCHIVED`.